2006Scape Client/src/main/java/ClientSettings.java

```diff
@@ -9,9 +9,39 @@
 public class ClientSettings {
     /**
      * @QoL
-     * Enables Server-Wide Snow In The Designated Month
+     * Hide roofs
      */
-    public static String SNOW_MONTH = ".12";
+    public static boolean HIDE_ROOFS = true;
+    /**
+     * @QoL
+     * Forces Server-Wide Snow floor
+     */
+    public static boolean SNOW_FLOOR_FORCE_ENABLED = false;
+    /**
+     * @QoL
+     * Toggles Server-Wide Snow floor
+     */
+    public static boolean SNOW_FLOOR_ENABLED = false;
```

Comment:
This is a duplicate of line 19

Reply:
Actually line 19 forces snow regardless of month, this one just determines if we want it in the snow month. They do serve distinct purposes.